torchsynth/parameter.py

```diff
@@ -108,10 +119,11 @@ class ModuleParameter(nn.Parameter):
 
     def __new__(
         cls,
-        value: T = None,
+        # TODO: REMOVEME
+        value: Optional[T] = None,
```

Comment:
Why remove value?

Reply:
If we want defaults that aren't 0.5, they should go into moduleparameterrange